Lib/ast.py

```diff
@@ -302,7 +302,7 @@ def _splitlines_no_ff(source):
 
 
 def _pad_whitespace(source):
-    """Replace all chars except '\f\t' in a line with spaces."""
+    """Replace all chars except '\\f\\t' in a line with spaces."""
```

Comment:
LGTM, but I prefer to use a raw string literal.

Reply:
I have no absolute preference. You can change it if you consider raw string as more suitable.

Reply:
@mpheath, thank you for your contribution to CPython and for your interest in improving the language and the documentation.  Serhiy's suggestion was asking if you could change it to a raw string in your PR and then he'll merge the change.

Reply:
Changed to raw string literal.